### ISSUE TITLE
Requests :)

### ISSUE THREAD

**Issue by ASH-II** [ZXDunny/SpecBAS#14]
Hi Paul

Could you add the FLIP,MIRROR and ROTATE to the PRINT?

10 PRINT MIRROR;"("
would give ")"

Would be a lot easier working with UDG's :)

**Comment by ZXDunny** [comment]
Actually, I'm working on proportional fonts right now (FZX files) so yes, I'll add this in.

MIRROR 1/0 - will mirror along the vertical axis.
FLIP 1/0 - will flip along the horizontal axis
ROTATE - this will take some doing. I assume that it will take a floating point value as an argument which is the rotation in radians (or degrees if you're in DEGREES mode)? The value will rotate from 0 (East facing) clockwise as it increases.

**Comment by ASH-II** [comment]
Yes if the rotation is not possible then a single roation on a UDG would help

5 DEGREES
10 FONT 6:ROTATE CHR$(128),90

**Comment by ZXDunny** [comment]
Rotation should be possible for any string in any direction. I can already do it, I just need to work it into PRINT.

**Comment by ASH-II** [comment]
ok so 
DEGREES:PRINT ROTATE 90;MIRROR 1;"blah" 
would be great.
👍

**Comment by ZXDunny** [comment]
Yes. that will be possible, as will:

DEGREES: PRINT ROTATE 45;MIRROR 1;PAPER 2; INK 6; SCALE 4,4;"Hi There!"

If I'm gonna do this properly.

**Comment by ZXDunny** [comment]
Leave it open until it's done.

**Comment by ASH-II** [comment]
k

**Comment by paperinho** [comment]
Scuse me, where are specbas_sysfiles.zip and demos.zip file ? Tnx

**Comment by ZXDunny** [comment]
> Scuse me, where are specbas_sysfiles.zip and demos.zip file ? Tnx

In the release zip: https://github.com/ZXDunny/SpecBAS/releases/tag/v.980 - download that and then replace the executables and dlls with the ones on the front page.

Though the demos are out of date and have been updated - you can find them on the demos repo at :
https://github.com/ZXDunny/SpecBAS-Demos

And use those instead.
